xcdat/bounds.py

```diff
@@ -260,12 +275,6 @@ def add_bounds(self, axis: CFAxisKey) -> xr.Dataset:
         ------
         ValueError
             If bounds already exist. They must be dropped first.
```

Comment:
```suggestion
```

This error is no longer thrown, the coordinates are just ignored if they already have bounds.